latex/method.tex

```diff
@@ -2,18 +2,20 @@ \section{Method}
 
 \subsection{Trajectory Representation}
 
-The general algorithm presented in this paper is agnostic to the choice of trajectory represention.
+The general algorithm presented in this paper is agnostic to the choice of trajectory representation.
 There is a large literature in trajectory optimization studying trajectory representation. \addref{trajectory representation}
 Here we choose to use shape-preserving cubic splines: the \texttt{PCHIP} spline type in Matlab.
 This choice of trajectory has a few useful properties: it is continuous in leaf position and velocity
 and it makes it simple to represent the trajectory using only the position at a few knot points.
 A piecewise linear spline would also be a reasonable choice.
 
 \todo{why cubic $>$ linear?  Would linear be faster?}
+\KvAcomment{If linear spline is indeed faster, I'd go for that. With low-order polynomials that shouldn't harm too much.}
```

Comment:
I think that I tried linear at some point and then moved on to cubic, but I don't remember why. I want to look back at the code to see if it have a good reason. If not, then maybe I will have the code to that instead.